server/src/main/java/org/opentosca/toscana/plugins/kubernetes/docker/mapper/MapperUtils.java

```diff
@@ -0,0 +1,27 @@
+package org.opentosca.toscana.plugins.kubernetes.docker.mapper;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Optional;
+
+import org.opentosca.toscana.model.capability.OsCapability;
+import org.opentosca.toscana.plugins.kubernetes.docker.mapper.model.DockerImageTag;
+
+public class MapperUtils {
+
+    public static final Comparator<DockerImageTag> TAG_COMPARATOR_MINOR_VERSION =
+        Comparator.comparing(DockerImageTag::toVersion);
+
+    public static boolean isAnyOptionalPresent(Optional... optionals) {
+        return Arrays.stream(optionals).anyMatch(Optional::isPresent);
+    }
+
+    public static boolean anythingSet(OsCapability capability) {
+        return isAnyOptionalPresent(
```

Comment:
It's the only use of `isAnyOptionalPresent`, so I suggest to merge both methods.